spacer/torch_utils.py

```diff
@@ -0,0 +1,130 @@
+"""
+This file contains a set of pytorch utility functions
+"""
+
+import torch
+import numpy as np
+from spacer import models
+from torchvision import transforms
+from collections import OrderedDict
+
+
+def gray2rgb(im):
+    w, h = im.shape
+    ret = np.empty((w, h, 3), dtype=np.uint8)
+    ret[:, :, 0] = im
+    ret[:, :, 1] = im
+    ret[:, :, 2] = im
+    return ret
+
+
+def transformation():
+    """
+    Transform an image or numpy array and normalize to [0, 1]
+    :return: a transformer which takes in a image and return a normalized tensor
+    """
+
+    transformer = transforms.Compose([
+        transforms.ToTensor(),
+    ])
+    return transformer
+
+
+def crop_patch(im, crop_size, scale, point_anns, offsets=None):
```

Comment:
This also looks similar to the one in `caffe_utils`. So move to shared util.

Reply:
`crop_patch()` moved to the shared `extract_features_utils.py`